actions/submit-signing-request/task.ts

```diff
@@ -292,10 +293,35 @@ export class Task {
 
             // if neither validationResult nor signingRequestId are present,
             // then the response might be not from the connector
+            core.error(`Unexpected response from the SignPath connector: ${JSON.stringify(response)}`);
             throw new Error(`SignPath signing request was not created. Please make sure that connector-url is pointing to the SignPath GitHub Actions connector endpoint.`);
         }
     }
 
+    private redirectConnectorLogsToActionLogs(logs: LogEntry[]): void {
+        if (logs && logs.length > 0) {
+            logs.forEach(log => {
+                switch (log.level) {
+                    case LogLevelDebug:
+                        core.debug(log.message);
+                        break;
+                    case LogLevelInformation:
+                        core.info(log.message);
+                        break;
+                    case LogLevelWarning:
+                        core.warning(log.message);
+                        break;
+                    case LogLevelError:
+                        core.error(log.message);
```

Comment:
Will logging to the error stream cause the workflow to actually fail? How does this interfere with the validation result?

Reply:
No, it will not. This has nothing to do with the validation result. It’s simply a way to convey additional information to the user—such as notifying them that, during validation, we noticed they have a policy file, so we will use it. Such a message is not part of the validation result; it’s just additional insight into what was happening in the connector.